src/middleware/src/storage/src/websql.js

```diff
@@ -7,20 +7,20 @@ import isFunction from 'lodash/isFunction';
 import isString from 'lodash/isString';
 const idAttribute = process.env.KINVEY_ID_ATTRIBUTE || '_id';
 const masterCollectionName = 'sqlite_master';
-const size = 5 * 1000 * 1000; // Database size in bytes
 let dbCache = {};
-let isSupported = undefined;
+let isSupported;
 
 export default class WebSQL {
-  constructor(name = 'kinvey') {
+  constructor(name = 'kinvey', size = 5000) {
```

Comment:
I initially thought to allow to pass a different size if needed with a default of 5mb. I can movie it back to a `CONST` if that makes more sense.